svgnative/CMakeLists.txt

```diff
@@ -64,10 +64,15 @@ CMAKE_DEPENDENT_OPTION(USE_CAIRO_EXAMPLE "Cairo example" TRUE "NOT LIB_ONLY;CAIR
 ################################
 if(CAIRO)
     find_package(PkgConfig)
-    include(FindCairo)
+    # include(FindCairo)
+    pkg_check_modules(CAIRO cairo)
     if(NOT CAIRO_FOUND)
-        set(CAIRO_INCLUDE_DIRS "/usr/include/cairo")
-        set(CAIRO_LIBRARIES "-lcairo")
+        message(FATAL_ERROR "Fatal error: Cairo port is requested, but CAIRO developers' files are missing.")
```

Comment:
So we reject building entirely now?

Reply:
@dirkschulze Yes, the old code was designed to try link cairo library even if it is not found by pkg-config. If a developer tries to build SVG Native Viewer with Cairo port, on the system without Cairo, the old code would cause an error. I think, reject building at the configuration phase might be easier to understand.

Or, should we disable Cairo port in the configuration phase (of course, some warning should be issued) is better? Other ports currently available (CoreGraphics, Skia, GDI+) have no check for the availability, so the discussion on Cairo availability would be the first discussion about such problem.

Reply:
Oh, I overlooked how "-DSKIA=ON" is handled. Currently, -DSKIA=ON is disabled automatically if the platform is not macOS...

Reply:
My current patch is still rejecting the building itself, if "-DCAIRO=ON" is given but cannot find working one. If you think "warn, disable and continue" is better, please comment.